src/main/java/dev/mlml/command/CommandRegistry.java

```diff
@@ -92,7 +92,44 @@ public static void executeCommand(Message message) {
 
         long cooldown = getCooldown(command, ctx.getMember().getId());
         if (cooldown > 0) {
-            message.reply(String.format("You must wait %d seconds before using this command again", cooldown / 1000))
+            long hours = cooldown / 3600000;
+            cooldown %= 3600000;
+
+            long minutes = cooldown / 60000;
+            cooldown %= 60000;
+
+            long seconds = cooldown / 1000;
+
+            StringBuilder sb = new StringBuilder("You must wait ");
+            if (hours > 0) {
+                sb.append(hours).append(" hour");
+                if (hours > 1) {
+                    sb.append("s");
+                }
+                sb.append(" ");
+            }
+            if (minutes > 0) {
+                sb.append(minutes).append(" minute");
+                if (minutes > 1) {
+                    sb.append("s");
+                }
+                sb.append(" ");
+            }
+            if (seconds > 0) {
+                sb.append(seconds).append(" second");
+                if (seconds > 1) {
+                    sb.append("s");
+                }
+                sb.append(" ");
+            }
+            if (hours + minutes + seconds == 0) {
+                sb.append("1 second ");
+            }
+
+            sb.append("before using this command again");
+            String cooldownReply = sb.toString().trim();
+
+            message.reply(cooldownReply)
```

Comment:
Don't we have a Utils class this would be perfect for? What there was another place we needed to pretty-print seconds?

Reply:
Yes, Utils class would be a better place for this.

To my knowledge there isn't another place, but moving into Utils would make sense regardless.

Reply:
Probably will be in the future 👍